pages/about/index.tsx

```diff
@@ -6,6 +6,7 @@ export default function Page() {
 
   useEffect(() => {
     router.push("/about/mission-and-goals")
+    router.push("/about/for-orgs")
```

Comment:
This will redirect all visits to /about to /about/for-orgs. If we should instead redirect to mission and goals, we can remove this line

Reply:
Got it, I think it should default to mission and goals, so I will remove this.